test/unit/core/selectors.ts

```diff
@@ -507,31 +505,184 @@ suite('selectors', ({ expect, stub }) => {
       expect(Selectors.pastPurchaseProductsBySku(state)).to.eql({ a: 4, b: 5, c: 8, });
     });
   });
+  describe('recommendations.pastPurchases', () => {
+    const skus = [{ a: 1 }];
+    const saytPastPurchases = [{ a: 1 }];
+    const query = 'hat';
+    const byId = { a: 1, b : 2, c: 3 };
+    const allIds = ['a', 'b', 'c'];
+    const navigations = { byId, allIds };
+    const sizes = {
+      selected: 1,
+      items: [10, 20, 30],
+    };
+    const page = {
+      current: 2,
+      last: 4,
+      sizes,
+    };
+    const sort = {
+      selected: 1,
+      items: ['a', 'b', 'c'],
+    };
+    const products = [1, 2, 3];
+    const pastPurchases = { skus, saytPastPurchases, products, query, navigations, page, sort };
+    const state: any = { data: { present: { pastPurchases } } };
 
-  describe('pastPurchases()', () => {
-    it('should return pastPurchases', () => {
-      const pastPurchases = { products: [{ a: 1 }]};
-      const state = { data: { present: { recommendations: { pastPurchases } } } };
+    describe('pastPurchases()', () => {
+      it('should return skus', () => {
+        expect(Selectors.pastPurchases(state)).to.eql(skus);
+      });
+    });
 
-      expect(Selectors.pastPurchases(<any>state)).to.eql([{ a: 1 }]);
+    describe('saytPastPurchases()', () => {
+      it('should return saytPastPurchases', () => {
+        expect(Selectors.saytPastPurchases(state)).to.eql(saytPastPurchases);
+      });
+    });
+
+    describe('pastPurchaseProducts()', () => {
+      it('should return pastPurchaseProducts', () => {
+        expect(Selectors.pastPurchaseProducts(state)).to.eql(products);
+      });
     });
-  });
 
-  describe('orderHistory()', () => {
-    it('should return orderHistory', () => {
-      const orderHistory = [{ a: 1 }];
-      const state = { data: { present: { recommendations: { orderHistory } } } };
+    describe('pastPurchaseQuery()', () => {
+      it('should return query', () => {
+        expect(Selectors.pastPurchaseQuery(state)).to.eql(query);
+      });
+    });
 
-      expect(Selectors.orderHistory(<any>state)).to.eql([{ a: 1 }]);
+    describe('pastPurchaseNavigation()', () => {
+      it('should return navigation object for given navigationId', () => {
+        expect(Selectors.pastPurchaseNavigation(state, 'b')).to.eql(2);
+      });
+    });
+
+    describe('pastPurchaseNavigations()', () => {
+      it('should return all navigations', () => {
+        stub(Selectors,'pastPurchaseNavigation').callsFake((s, id) => id);
+
+        expect(Selectors.pastPurchaseNavigations(state)).to.eql(allIds);
+      });
+    });
+
+    describe('pastPurchaseSelectedRefinements()', () => {
+      it('should call getSelected with allIds and return it\'s result', () => {
```

Comment:
avoid using apostrophes in the `it()` names, could just be `"should call getSelected with allIds and return the result"`